piksi_tools/console/update_downloader.py

```diff
@@ -23,19 +23,29 @@ def __init__(self):
     self.index = jsonload(f)
     f.close()
 
-  def download_stm_firmware(self):
+  def download_stm_firmware(self, hwrev):
     try:
-      url = self.index['piksi_v2.3.1']['stm_fw']['url']
+      url = self.index[hwrev]['stm_fw']['url']
       filepath = self._download_file_from_url(url)
     except KeyError:
       raise KeyError("Error downloading firmware: URL not present in index")
     except URLError:
       raise URLError("Error: Failed to download latest NAP firmware from Swift Navigation's website")
```

Comment:
can we fix the incorrect message here? (this is in the STM firmware and the message says NAP)